docs/parameters.md

```diff
@@ -0,0 +1,61 @@
+# Storage Class Parameters
+The AWS FSx for OpenZFS CSI Driver supports the dynamic provisioning of both FSx for OpenZFS file systems AND child volumes.
+
+To do this, the driver accepts **TWO** forms of storage classes: one for provisioning new file systems, and one for provisioning new child volumes.
+
+## File System Storage Class Parameters
```

Comment:
nit: I feel like `File System Parameters` is explicit enough, and easier to read through